### DISCUSSION TITLE
small fonts support on Linux

### DISCUSSION THREAD

**Discussion by jimkring** [ni/labview-icon-editor#421] [Announcements]
I need to confirm if this is still an issue, but there have been some challenges using small fonts on Linux.

**Comment by rtollert** [comment]
It is; my attempt at elaborating concrete solutions was #7.

**Comment by jimkring** [comment]
@svelderrainruiz This is marked as completed, but there is no pull request or Milestone associated. I'm curious if this is actually complete completed.

**Comment by svelderrainruiz** [comment]
@jimkring this was not completed. This issue should have been converted into a discussion instead of closing it. Doing that now.

**Comment by jimkring** [comment]
Consistent small fonts in Icons accross platforms is one of the major reasons (IMO) that use of LV on Linux and macOS is hindered. It is effectively impossible to create “good” code on Linux and macOS, where good means “also has good icons”. But, it’s not just that the icons aren’t good—they are horrible.  This needs to be a high priority. Small Fonts 8 has been a labview standard since forever. Why is this (or equivalent pixel font) not built into LabVIEW?

**Comment by jimkring** [comment]
I have drafted a PR that is a start to addressing this need: https://github.com/ni/labview-icon-editor/pull/433
